StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -14,6 +14,10 @@ import Foundation
     /// - Note: The PaymentIntent or SetupIntent you create on your server must have the same values or the payment/setup will fail.
     struct IntentConfiguration {
 
+        /// Pass this into `intentCreationCallback` to force PaymentSheet to show success and dismiss.
+        /// - Note: Only for advanced users, not required for most integrations.
+        public static let FORCE_SUCCESS = "FORCE_SUCCESS"
```

Comment:
Noticed this is already under `@_spi(ExperimentalPaymentSheetDecouplingAPI)`, not sure we need to make it under a different SPI based on our discussion in the sync today @yuki-stripe.

Reply:
Under STP for now